lib/block.ex

```diff
@@ -2,6 +2,7 @@ defmodule UltraDark.Blockchain.Block do
   alias UltraDark.Blockchain.Block
   alias UltraDark.Utilities
   alias UltraDark.Transaction
+  alias Decimal, as: D
```

Comment:
lets use this library as 'Decimal' instead of 'D', as its not as descriptive of whats happening

Reply:
That'd make a lot of the code really verbose. Also, the documentation of the library suggests you alias the library as `D`

Reply:
Fair enough. Could you update with the other comment I made? It'll be ready for merge once that happens

Reply:
Okay, I've done that now